app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -57,17 +60,14 @@
              "\r\n\r\n"
              (->b64 body))))))
 
-(defn- send-email!*
-  "Invoke email sending"
-  [msg]
-  (aws/invoke
-   @email
-   {:op :SendRawEmail
-    :request {:RawMessage {:Data (.getBytes (raw-message msg) "UTF-8")}}}))
+(defn send-email-smtp* [msg]
+  (println (str "email server " email-server))
+  (println (str "message " msg))
+  (println (postal.core/send-message email-server msg)))
 
 (defn send-email! [msg]
   (let [prefix (environment/config-value :email :subject-prefix)
         msg (if prefix
               (update msg :subject #(str prefix " " %))
               msg)]
-    (send-email!* msg)))
+    (send-email-smtp* msg)))
```

Comment:
Take server config from environment in send-email! and pass it to send-email-smtp* function.

Can't use config defined in namespace

Reply:
Also should update from master as this has changed there